.github/workflows/rosa-cluster-delete.yml

```
@@ -20,8 +23,14 @@ jobs:
           aws-default-region: ${{ vars.AWS_DEFAULT_REGION }}
           rosa-token: ${{ secrets.ROSA_TOKEN }}
 
+      - name: Determine cluster name
+        run: |
+          CLUSTER_NAME_USER_INPUT=${{ inputs.clusterName }}
+          ACTUAL_CLUSTER_NAME=${CLUSTER_NAME_USER_INPUT:-"gh-${{ github.triggering_actor }}"}
```

As mentioned above default should be based on owner of fork, not triggering actor:

```suggestion
          ACTUAL_CLUSTER_NAME=${CLUSTER_NAME_USER_INPUT:-"gh-${{ github.github.repository_owner }}"}
```